.tav.yml

```diff
@@ -362,8 +362,12 @@ tedious:
 
 cassandra-driver:
   # 3.1.0 is broken
-  versions: '>=3 <3.1.0 || >3.1.0 <5'
-  commands: node test/instrumentation/modules/cassandra-driver/cassandra-driver.test.js
+  - versions: '>=3 <3.1.0 || >3.1.0 <4.8.0'
+    commands: node test/instrumentation/modules/cassandra-driver/cassandra-driver.test.js
+  # v4.8.0 advertise a min-supported Node.js version of 18.
+  - versions: '>=4.8.0 <5'
```

Comment:
nit: Given this on the jira issue:

> For what it’s worth support for node.js 16.x was officially removed with the 4.7.0 release in accordance with this policy.  This was discussed in a [message sent to the mailing list](https://groups.google.com/a/lists.datastax.com/g/nodejs-driver-user/c/nHCkJuNxl_s/m/z-JrsLwVAAAJ) although I think we did miss the “engines” update until this release.

should we get 4.7.0 as the change point rather than 4.8.0?